### PR TITLE
#178 - fix onboarding parent/child logic

### PR DESCRIPTION
**Pull request by KennyKeni**
## PR Title

Issue Number(s): #178 

What does this PR change and why?
Pass UserType to completion step

### Checklist

- [x] Requirements have been implemented
- [x] Acceptance criteria is met
- [x] Database schema docs have been updated or are not necessary
- [x] Code follows design and style guidelines
- [x] Commits follow guidelines (concise, squashed, etc)
- [x] Relevant reviewers (Senior Dev/EM/Designers) have been assigned to this PR

### Critical Changes

- Database change/migration to run
- Environment config change
- Breaking API change

### Testing

Enumerate steps to test the functionality of your ticket. This should include edge cases and testing of error handling, if applicable.

### CODE REVIEW

**Comment by rgodha24** [comment]
@KennyKeni is this done can i merge

File: src/components/onboarding/steps/CompletionStep.tsx [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
+  userType,
   currentStep,
   onBack,
   onComplete,
 }: CompletionStepProps) {
+  const medicationLine = `You've accepted the terms. Now let's set up ${userType === "parent" ? "your child's" : "your"} medications.`;
+  const petChoiceLine =
+    userType === "parent"
+      ? "Pass the device to your child so they can pick their pet to begin the fun!"
+      : "Now you can choose your pet to begin the fun!";
```
[the remark this answers is not in the document]
Comment (KennyKeni) [reply src/components/onboarding/steps/CompletionStep.tsx]: I think its actually the useState being reset because we need to navigate to the medication page, this userType is always undefined during completion step
